packages/dashboards/tests/acceptance/dashboards-test.js

```diff
@@ -380,9 +380,14 @@ module('Acceptance | Dashboards', function(hooks) {
   });
 
   test('Failing to save a new widget', async function(assert) {
-    assert.expect(6);
+    assert.expect(8);
 
-    server.patch('/dashboards/1', () => new Response(500));
+    server.patch('/dashboards/1', (_, request) => {
+      assert.equal(request.requestHeaders['content-type'], 'application/vnd.api+json');
```

Comment:
Could we give some messages to the assert statements?

Reply:
done